src/main/java/Ui.java

```diff
@@ -0,0 +1,253 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    protected Scanner scan = new Scanner(System.in);
+    protected TaskList tL = new TaskList();
+    protected Storage store = new Storage(Storage.file);
+
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+
+    }
+
+    /**
+     * Prints the greeting at the initiation of the chat bot.
+     */
+    public void greeting() {
+        String logo = "     ____        _        \n" +
+                      "    |  _ \\ _   _| | _____ \n" +
+                      "    | | | | | | | |/ / _ \\\n" +
+                      "    | |_| | |_| |   <  __/\n" +
+                      "    |____/ \\__,_|_|\\_\\___|\n";
+
+        // Prints out greeting of the chat bot.
+        printLine();
+        printIndent();
+        System.out.println("Hello! My name is \n" + logo + "\n" +
+                "    What can I do for you? \n");
+        printIndent();
+        System.out.println("I can only do these functions for now: \n \n" +
+                "    Todo \n" + "        Eg. todo __(task)__\n" +
+                "    Event \n" + "        Eg. event __(task)__ /at _(dd/MM/yyyy)_(hhmm)__\n" +
+                "    Deadline \n" + "        Eg. deadline __(task)__ /by _(dd/MM/yyyy)_(hhmm)__\n" +
+                "    Delete \n" + "        Eg. delete __(number)__ or delete all\n" +
+                "    Done \n" + "        Eg. done __(number)__\n" +
+                "    List \n");
+        printIndent();
+        System.out.println("Ill be adding in more features soon! Please be patient! :)");
+        printLine();
+    }
+
+    /**
+     * Scans the input from the user and does the respective
+     * command based on the input. Prints out error messages as well.
+     */
+    public void nextCommand() {
+        while (scan.hasNext()) {
```

Comment:
I think we should try to use the Parser class you have created to parse the input first! This way we can abstract out the parsing functions and so the UI class can just be concerned with the UI tasks.